### ISSUE TITLE
Allow to display crash reports locally before sending them to the reporter 

### ISSUE THREAD

**Issue by SamuAlfageme** [owncloud/client#6055]
This would be super-useful since sentry (or socorro?) does not seem to be updated in real time and also will complement the info users can provide upon crashes. 

cc/ @dschmidt

**Comment by guruz** [comment]
I don't think the symbols are available locally, they are added by sentry/socorro

**Comment by SamuAlfageme** [comment]
@guruz yeah, that's what I thought 😕 - maybe we can offer the chance to "download an additional symbols package" like e.g. Firefox does. Dunno how hard this might be, though.
